pkg/controllers/management/wrangler.go

```diff
@@ -17,7 +17,7 @@ func RegisterWrangler(ctx context.Context, wranglerContext *wrangler.Context, ma
 	if features.Legacy.Enabled() {
 		k3sbasedupgrade.Register(ctx, wranglerContext, management, manager)
 		eks.Register(ctx, wranglerContext, management)
+		clusterupstreamrefresher.Register(ctx, wranglerContext)
```

Comment:
This will need to move outside of legacy block in 2.6. But we may need to leave inside block just to test.

Reply:
How do we decide if this is legacy?

Reply:
Legacy is system-charts